setup.py

```diff
@@ -53,30 +47,31 @@
         if len(data) == 1:
             VERSION = data[0]
         elif data and data[0].lower() == "release":
-            VERSION = "%s.%s"%(data[1], data[2])
+            VERSION = "%s.%s" % (data[1], data[2])
         else:
-            VERSION = "%s.%s"%(data[0], data[1])
+            VERSION = "%s.%s" % (data[0], data[1])
     except:
         raise RunTimeError("git tags must be in the form release-x.y.z or simply x.y.z")
 else:
-  VERSION="2.2.0" # hardcode version
+    VERSION = "2.2.0"  # hardcode version
 
-setup(name='descriptastorus',
-      version=VERSION,
-      description='Descriptor storage and molecular file indexing',
-      author='Brian Kelley',
-      author_email='brian.kelley@novartis.com',
-      url='https://bitbucket.org/novartisnibr/rdkit-descriptastorus/',
-      install_requires=['pandas_flavor'],
-      test_suite='nose.collector',
-      tests_require=['nose', 'pandas_flavor'],
-      include_package_data=True,
-      entry_points={
-          'console_scripts': [
-              'storus = descriptastorus.cli.storus:main',
-              'storus-validate = descriptastorus.cli.validate:main',
-              ]
-          },
-                                                            
-      packages = find_packages())
+setup(
+    name="descriptastorus",
+    version=VERSION,
+    description="Descriptor storage and molecular file indexing",
+    author="Brian Kelley",
+    author_email="brian.kelley@novartis.com",
+    url="https://bitbucket.org/novartisnibr/rdkit-descriptastorus/",
+    install_requires=["pandas_flavor", "rdkit"],
```

Comment:
Only edited this line and removed the ```import rdkit``` try-except up top. The rest is black formatter.